pkg/deploy/defaults.go

```diff
@@ -28,6 +28,7 @@ const (
 	DefaultIngressClass              = "nginx"
 	DefaultPluginRegistryUrl         = "https://che-plugin-registry.openshift.io"
 	DefaultUpstreamPluginRegistryUrl = "https://che-plugin-registry.openshift.io/v3"
+	DefaultDevfileRegistryUrl        = "https://che-devfile-registry.openshift.io"
```

Comment:
will these eventually change to be part of the Che deployment, once the registries are local to the deployment rather than shared instances on openshift.io?

Reply:
Yes, I will change that in a next PR